### DISCUSSION TITLE
New way to render from ERB?

### DISCUSSION THREAD

**Discussion by cmer** [yippee-fun/phlex#572] [Help] [answered]
I'm trying to upgrade from an old version of Phlex (0.5.3 I believe). I now get this error when trying to render my Phlex component from erb: "must implement :to_partial_path". What's the new way to render? I use: <%= render Students::StatusBadgeComponent.new(program_enrollment:) %>

**Comment by joeldrapper** [comment, marked as answer]
The Rails integration was extracted into `phlex-rails` since `0.5.3`. You should use that gem instead. It depends on `phlex`, so you can have just `phlex-rails` in your Gemfile.
